recipe/meta.yaml

```diff
@@ -34,8 +34,8 @@ test:
   source_files:
     - cotire_src
   requires:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}                                                                         # [unix]
+    - {{ compiler('cxx') }}                                                                       # [unix]
```

Comment:
Trying this as a workaround for issue ( https://github.com/conda/conda-build/issues/4605 ) in the hope of getting more info on CI about what is failing (since that is currently obfuscated). That said, expect this will still fail as compilers are needed here.